fedireads/incoming.py

```diff
@@ -118,7 +119,7 @@ def handle_follow(activity):
     # figure out who the actor is
     user = get_or_create_remote_user(activity['actor'])
     try:
-        request = models.UserFollowRequest.objects.create(
+        relationship, _ = models.UserFollowRequest.objects.get_or_create(
```

Comment:
I don't know if the get_or_create is correct here. (If it is, the code and comment below handling the duplicate needs to change.)

Reply:
you're right! this should stay as `create`